src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -1029,6 +1029,20 @@ public IServer GetServer(EndPoint? endpoint, object? asyncState = null)
             return new RedisServer(this, server, asyncState);
         }
 
+        /// <summary>
+        /// Obtain configuration APIs for all servers this multiplexer.
```

Comment:
"for all servers IN this multiplexer"? or "for all servers this multiplexer IS CONNECTED TO"? 

(this string also appears in IConnectionMultiplexer.cs)

Reply:
Good catch - fixed!